scripts/install.js

```diff
@@ -236,6 +262,11 @@ function bootstrap(cb) {
 			});
 		},
 		function (callback) {
+			// For some reason, when using the --prefix flag, an etc folder is added.
```

Comment:
Any idea where this is coming from? Is that NPM, and if so, is there an issue for this?